### ISSUE TITLE
Module to manage routes

### ISSUE THREAD

**Issue by neikei** [IBM/ansible-power-aix#175]
**Is your feature request related to a problem? Please describe.**
It would be a great enhancement to be able to manage routes via Ansible. Especially for admins of linux systems aix could become more accessible.

**Describe the solution you'd like**
I would like to manage temporary routes (command: route) and persistent routes (command: chdev -l inet0 ...) via Ansible.

**Describe alternatives you've considered**
Managing routes via shell-module and a lot of changed_when / failed_when conditions.

**Additional context**

**Comment by Architecttheday** [comment]
+1

**Comment by pvtorres** [comment]
@neikei, The persistent routes can be managed via the ibm.power_aix.devices module. But I will bring the the request to the team. 
Example:

  - name: Add a static route
     ibm.power_aix.devices:
        device: inet0
       state: available
       attributes:
          route: ",-hopcount,0,,,,,,-static,0,0.0.0.0"

  - name: Remove a static route
     ibm.power_aix.devices:
        device: inet0
        state: available
        attributes:
           delroute: ",-hopcount,0,,,,,,-static,0,0.0.0.0"

**Comment by neikei** [comment]
Has the topic been evaluated in the meantime?

**Comment by nitismis** [comment]
yeah we did evaluate it and posted in our automation platform for voting, so that, we can prioritize it. Because of lack of enough voting, we could not do it. If this is important to you, then may be we need to prioritize this anyway. I am going to discuss this with team and update you.

**Comment by nitismis** [comment]
This will be part of our year-end release in December.

**Comment by nitismis** [comment]
New route module has been added now. Please raise new issues if you want further enhancement.
